packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -263,3 +327,62 @@ export function evalBlobsAndTrees(snapshot: IOdspSnapshot) {
     }
     return { numTrees, numBlobs, encodedBlobsSize, decodedBlobsSize };
 }
+
+export function isRedeemSharingLinkError(odspResolvedUrl: IOdspResolvedUrl, error: any) {
+    const errorType = error.errorType;
+    if (odspResolvedUrl.sharingLinkToRedeem !== undefined
+        && (errorType === DriverErrorType.authorizationError
```

Comment:
Please add a test that typeof error === "object" && error !== null (there is some helper function somewhere)

Reply:
I don't think there is a helper function. But added the check.